### PR TITLE
feat: angular 15 release

### PR DESCRIPTION
**Pull request by gsontag**
BREAKING CHANGES: upgrade to `@angular/core 15.0.x` is required!

### CODE REVIEW

File: projects/moment/package.json [hunk 1]
```diff
@@ -5,11 +5,11 @@
...
-    "@angular/common": "^14.0.5",
-    "@angular/core": "^14.0.5",
-    "@angular/material": "^14.0.4",
-    "@angular/material-moment-adapter": "^14.0.4",
+    "@angular/cdk": "^15.0.0",
+    "@angular/common": "^15.0.1",
+    "@angular/core": "^15.0.1",
+    "@angular/material": "^15.0.0",
+    "@angular/material-moment-adapter": "^15.0.0",
     "@mat-datetimepicker/core": "^10.0.0"
```
Comment (bbortt) [inline projects/moment/package.json]: I think this bump will not happen automatically!

```suggestion
    "@mat-datetimepicker/core": "^11.0.0"
```

File: src/styles.scss [hunk 2]
```diff
@@ -1,22 +1,29 @@
...
+    typography: mat.define-typography-config(),
+    density: 0,
+  )
+);
+
+@include mat.all-component-themes($theme);
 
 // Use this import in a real project instead
-//@import '~@mat-datetimepicker/core/datetimepicker/datetimepicker-theme.scss';
+//@import '@mat-datetimepicker/core/datetimepicker/datetimepicker-theme.scss';
```
Comment (bbortt) [inline src/styles.scss:26]: sorry, I know this is definitely something I should do, but I do not find any time. would you mind updating the [theming part of the `README.md`](https://github.com/kuhnroyal/mat-datetimepicker#theming) as well?

**Review by bbortt — CHANGES_REQUESTED** [review]

**Comment by bbortt** [comment]
closes #403.

**Comment by gsontag** [comment]
Made a workaround in angular.json for empty test suite because "failOnEmptyTestSuite: false" seems to be ignored

File: projects/moment/package.json [hunk 3]
```diff
@@ -1,16 +1,16 @@
 {
   "name": "@mat-datetimepicker/moment",
-  "version": "10.0.10",
+  "version": "11.0.0",
```
Comment (bbortt) [inline projects/moment/package.json]: sorry if I confused you with [my earlier comment](https://github.com/kuhnroyal/mat-datetimepicker/pull/404#discussion_r1031869570). this version will be bumped (see the [`.releaserc.js`](https://github.com/kuhnroyal/mat-datetimepicker/blob/release/.releaserc.js#L35) configuration)! just not the `peerDependencies`. please revert just this line.

File: projects/core/package.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mat-datetimepicker/core",
-  "version": "10.0.10",
+  "version": "11.0.0",
```
Comment (bbortt) [inline projects/core/package.json]: this version will be bumped (see the [`.releaserc.js`](https://github.com/kuhnroyal/mat-datetimepicker/blob/release/.releaserc.js#L35) configuration)! please revert this change.

**Review by bbortt — CHANGES_REQUESTED** [review]
sorry, I might have confused you with my previous comments.. otherwise /lgtm!

File: projects/core/package.json (hunk 4 above)
Comment (gsontag) [reply projects/core/package.json]: ah, ok :p not used to releaserc

**Review by bbortt — APPROVED** [review]
thanks so much, I've checked it out too and everything looks fine to me. let's ship this 🚀

**Pull request merged**